src/content/getting-started/persistent-volumes/vmware-cloud-director/index.md

```diff
@@ -0,0 +1,108 @@
+---
+linkTitle: PVs on VMware Cloud Director
+title: Using persistent volumes on VMware Cloud Director
+description: Tutorial on how to use dynamically provisioned Persistent Volumes on a cluster running on VMware Cloud Director.
+weight: 10
+menu:
+  main:
+    identifier: gettingstarted-persistentvolumes-clouddirector
+    parent: gettingstarted-persistentvolumes
+aliases:
+  - /guides/using-persistent-volumes-on-clouddirector/
+  - /ui-api/observability/prometheus/persistent-volumes/clouddirector/
+user_questions:
+  - How can I use persistent volumes in my VMware Cloud Director clusters?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-rocket
+last_review_date: 2023-10-05
+---
+
+If your cluster is running on VMware Cloud Director (VCD), it comes with a dynamic storage provisioner for Named Disks. This enables you to store data beyond the lifetime of a Pod into virtual disks.
+
+## Storage Classes
+
+Your Kubernetes cluster will have a default Storage Class `csi-vcd-sc-delete` deployed, which will automatically get selected if you do not specify the Storage Class in your Persistent Volumes. Another Storage Class `csi-vcd-sc-retain` is also created with `reclaimPolicy: Retain` which you must explicitely specify to use.
+
+As a Cluster Admin you can create additional Storage Classes or edit the default class to use different types of VMware Storage Profiles or for instance, add encryption.
+
+For this you just need to create (or edit) [Storage Class objects](https://kubernetes.io/docs/concepts/storage/persistent-volumes).
+
+## Creating Persistent Volumes
+
+The most straight forward way to create a Persistent Volume is to create a [Persistent Volume Claim object](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims), which will automatically create a corresponding Persistent Volume (PV) for you.
+
+Alternatively, to be able to set more specific parameters on your PV you can first create a [Persistent Volume object](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistent-volumes) and then claim that PV using a Persistent Volume Claim (PVC).
+
+Under the hood, the Dynamic Storage Provisioner will take care that a corresponding Named Disk with the correct parameters is created in the VMware Cloud Director OVDC.
+
+The Named Disk and its data will persist as long as the corresponding PV resource exists. Deleting the resource will also delete the corresponding Named Disk, which means that all stored data will be lost at that point.
+
+_Note: Do not delete virtual machines that have a Named Disk attached or it will be left in an inconsistent state and the VCD admins will need to intervene to clean it up._
```

Comment:
```suggestion
__Note__: Do not delete virtual machines that have a Named Disk attached or it will be left in an inconsistent state and the VCD admins will need to intervene to clean it up.
```